pwd_guess.py

```diff
@@ -826,6 +830,13 @@ def __init__(self, pwd_list, config=ModelDefaults(), multi_gpu=1):
         self.model_to_save = None
         self.multi_gpu = multi_gpu
         self.pwd_list = pwd_list
+        self.cumulative_chunks = 0
```

Comment:
This variable only seems to be used inside  of train_model_generation.  Could it be moved as a local variable in that function?

Reply:
This function is aggregating chunk counts across different function counts so it can't be a local variable within that function

Reply:
OK, that's fine then.